style-guidelines/Swift.md

```diff
@@ -141,8 +141,14 @@ else {
 
 ## Classes and Structs
 
-Unless you require functionality that can only be provided by a class (like identity or deinitializers), implement a struct instead.
+Unless you require functionality that can only be provided by a class, implement a struct instead.
 
+Additional capabilities of classes:
+
+- Inheritance: Enables one class to inherit the characteristics of another
+- Type casting: Enables you to check and interpret the type of a class instance at runtime
```

Comment:
First 2 punts can be solved for structs as well as : 
- Struct con conform to a protocol. Similar to type casting and checking
- Instead of Inheritance you have have composition (which has been proved to be better way)

With those 2 punks I agree. If you need it than you got for classes. 
- Deinitializers
- Reference counting: